client/src/components/products/ProductItemList/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useMemo, useState } from 'react'
+import InfiniteScroll from 'react-infinite-scroll-component'
+import { useSelector } from 'react-redux'
+
+import ProductItem from '$components/products/ProductItem'
+import { fetchProducts, isProductsLoadingSelector, productsSelector } from '$stores/product'
+
+import { useAppDispatch } from '$stores'
+
+export default function ProductItemList() {
+  const dispatch = useAppDispatch()
+  const isLoading = useSelector(isProductsLoadingSelector)
+  const products = useSelector(productsSelector)
+  const [count, setCount] = useState(10)
+
+  const displayedProducts = useMemo(() => products.slice(0, count), [products, count])
+
+  useEffect(() => {
+    dispatch(fetchProducts())
+  }, [dispatch])
+
+  if (isLoading) {
+    return null
+  }
```

Comment:
```suggestion
  if (isLoading) {
    return <div>....loading</div>
  }
```
loading 상태는 null보다 간단한 문구라도 주는게 좋을것 같아요. 테스트할때 외부 요청이 이루어지고있나 잠깐 고민했습니다